modules/javafx.graphics/src/main/resources/com/sun/prism/es2/glsl/main1Light.frag

```diff
@@ -88,9 +88,9 @@ void main()
 
         float power = specular.a;
 
-        float att = 1.0 / (lights[0].attn.x + lights[0].attn.y * dist + lights[0].attn.z * dist * dist);
-        d = clamp(dot(n, l), 0.0, 1.0) * (lights[0].color).rgb * att;
-        s = pow(clamp(dot(-refl, l), 0.0, 1.0), power) * lights[0].color.rgb * att;
+        float attenuatedColor = (lights[0].color).rgb / (lights[0].attn.x + lights[0].attn.y * dist + lights[0].attn.z * dist * dist);
```

Comment:
This fails (on macOS) at shader compilation time with:

```
Shader compile log: ERROR: 0:325: Incompatible types in initialization (and no available implicit conversion)
ERROR: 0:326: Use of undeclared identifier 'attenuatedColor'
...
```

That should be `vec3` (same for the other two shader files)